### PR TITLE
Add info about Emlid Reach RTK Receiver

### PR DESCRIPTION
**Pull request by i-smirnov**

### CODE REVIEW

**Comment by hamishwillee** [comment]
@rmackay9 @magicrub This needs a subedit (which I don't have time for) but sounds pretty cool. Can you review?

My only other comment is that this is a positioning topic. Arguably it should go in the [GPS/Compass](https://github.com/ArduPilot/ardupilot_wiki/blob/master/common/source/docs/common-positioning-landing-page.rst) landing page - which needs to be re-renamed "Positioning (GPS, Compass, RTK)" rather than the optional hardware. What do you think?

**Comment by i-smirnov** [comment]
@hamishwillee, I think it's a good idea to move Reach topic to GPS/Compass page. I'll be glad to make required edits depending on your decision.

**Comment by hamishwillee** [comment]
@rmackay9 Can you comment? @i-smirnov  I need Randy to comment because we explicitly changed the name the "GPS/Compass" from the more generic "Positioning" so I need he feedback.

**Comment by i-smirnov** [comment]
@rmackay9, we are really looking forward to add Reach in ardupilot documentation. Please tell what you think. Thank you!

File: common/source/docs/common-reach-rtk-receiver.rst [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+Reach is an RTK GNSS receiver for applications when your standard GPS with several meters accuracy just won't cut it. It relies on RTK (real-time kinematics) technology to deliver centimeter level accuracy.
+
+RTK was here for a long time, used mostly by surveyors and unaffordable to hobbyists and makers. If you needed centimeter precise positioning you had to spend thousands of dollars on an RTK system. Reach changed that.
+
+Reach runs open-source RTK processing software called RTKLIB written by Tomoji Takasu. Previously a computer was required to run RTKLIB, but now all RTKLIB features are available directly on Reach.
+
+For further information read `Reach documentation <http://docs.emlid.com/reach/introduction/>`__.
+
+APM integration
```
Comment (magicrub) [inline common/source/docs/common-reach-rtk-receiver.rst]: Please reference the project as Ardupilot instead of APM.

**Comment by hamishwillee** [comment]
@i-smirnov And yes please ... move under GPS.

**Comment by i-smirnov** [comment]
@hamishwillee @rmackay9, I updated the PR. 
* Reach topic moved to GPS
* APM references replaced by Ardupilot

**Comment by hamishwillee** [comment]
Thanks @i-smirnov - I hope to look at this Monday.

**Comment by hamishwillee** [comment]
@i-smirnov - Thanks Ivan. FYI it is ArduPilot but I am happy to tidy that up after the fact. 

Small problem is that the images are 500K, and could easily be reduced below 100K simply by changing to a JPG. This doesn't really matter on a single case, but every image added means an increase for anyone wanting to clone the repo ... and increases the download time. Any chance you could reduce them? I would then be happy to merge.

**Comment by i-smirnov** [comment]
Thanks @hamishwillee. I reduced size of the images.

**Comment by hamishwillee** [comment]
Thanks very much! Merged.

**Comment by hamishwillee** [comment]
... and I've just given it a minor subedit.

**Pull request merged**